packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -181,6 +181,18 @@ async function unpack(prepareOptions: PrepareApplicationStageDirectoryOptions, o
       return
     }
     await executeAppBuilder(["unpack-electron", "--configuration", JSON.stringify([options]), "--output", appOutDir, "--distMacOsAppName", distMacOsAppName])
+    if (packager.config.downloadAlternateFFmpeg) {
+      log.info("downloading and unpacking alternate FFmpeg")
+      const ffmpegOptions = Object.assign({}, options)
+      ffmpegOptions.customFilename = `ffmpeg-v${options.version}-${options.platform}-${options.arch}.zip`
+      let ffmpegLocation = path.join(appOutDir, "..")
+      if (!isEmptyOrSpaces(process.env.ELECTRON_BUILDER_CACHE)) {
+        ffmpegLocation = process.env.ELECTRON_BUILDER_CACHE
+      }
+      ffmpegOptions.cache = ffmpegLocation
```

Comment:
Hmm, I was basing this off of the app-builder arguments, where it expects the `cache` property in the config to be a directory.

https://github.com/develar/app-builder/blob/4e2aa6a12e2bc3d31ec0d01d661fb3a4d65248ff/pkg/electron/electronDownloader.go#L20

and

https://github.com/develar/app-builder/blob/4e2aa6a12e2bc3d31ec0d01d661fb3a4d65248ff/pkg/electron/electronDownloader.go#L62

However, I mistakenly read the `ELECTRON_CACHE` as `ELECTRON_BUILDER_CACHE` and defaulted to that if it were explicitly specified other than the default to take advantage of caching this ffmpeg download between runs if users had specified the ELECTRON_CACHE folder.

I was really hoping to use a `getCacheFolder` method that I could call and not have to specify any specific directory, since we a) probably want to keep this download cached like we do electron distributables and b) we need to know where it ends up after downloading so that we can unzip it.

Perhaps the caching is a premature optimization and it's easiest and cleanest to just keep it up in `path.dirname(appOutDir)` for now without the caching logic, since this zip is ever only around 1mb.

Reply:
Gotcha gotcha. Yes, great callout as well, I think the caching is an unnecessary optimization for now when it's that is only 1mb.

Just to double check, is this working in your personal/test project? 
Have you given any thought on how we could integration test this? I'll continue thinking from my end. Is there a way for us to detect whether a new `FFmpeg` was downloaded or not? As we do have a post-packaging hook for verifying app contents.